sites/en/installfest/configure_git.step

```diff
@@ -1,9 +1,9 @@
 message "(If you used RailsInstaller on Windows then you should already have `user.name` and `user.email` configured.)"
 
-console <<-BASH
-git config --global user.name "Your Actual Name"
-git config --global user.email "Your Actual Email"
-BASH
+message "In the next commands, don't copy-paste the whole thing. Use the name and email address you usually use. Don't use a username (like your github username)."
```

Comment:
Is "don't copy-paste the whole thing" a little abrupt? Is there better phrasing?